garage/envs/wrappers/stack_frames.py

```diff
@@ -0,0 +1,58 @@
+"""Stack frames wrapper for gym.Env."""
+from collections import deque
+
+import gym
+from gym.spaces import Box
+import numpy as np
+
+
+class StackFrames(gym.Wrapper):
+    """
+    Stack frames wrapper.
+
+    gym.Env wrapper to stack multiple frames.
+    Useful for training feed-forward agents on dynamic games.
+    """
+
+    def __init__(self, env, n_frames):
+        """
+        Stack frames wrapper.
+
+        Args:
+            env: gym.Env to wrap.
+            n_frames: number of frames to stack.
+
+        Raises:
+            ValueError: If observation space shape is not 2.
```

Comment:
Would a shape of `(120, 240)` be valid? Isn't that a single-channel 2D image?

Reply:
It would be valid.

Reply:
alright